core/Parsing.Documents.fs

```diff
@@ -150,7 +150,7 @@ let mutable languages = [
               line "#"
             ]
         )
-    lang "HTML" "htmlx|vue" ".htm|.html|.vue"
+    lang "HTML" "htmlx|vue|erb" ".htm|.html|.vue|.html.erb"
```

Comment:
Hi and thanks for this!

```suggestion
    lang "HTML" "htmlx|vue|erb" ".htm|.html|.vue"
```

After having tested it manually, unfortunately adding the `.html.erb` extension doesn't do anything. This is because vanilla VS Code already sees all `.erb` files as Ruby files, and this won't change that.

The user will need to have a language extension installed that defines the "erb" language and sets the file extension associations accordingly, and that will in turn get picked up here.

Alternatively they can add their own **Files: Associations** user setting, to configure (.html).erb as "html" files instead.